cas-server-core-authentication/src/main/java/org/jasig/cas/authentication/handler/DefaultPasswordEncoder.java

```diff
@@ -24,9 +29,11 @@
     private static final int HEX_RIGHT_SHIFT_COEFFICIENT = 4;
     private static final int HEX_HIGH_BITS_BITWISE_FLAG = 0x0f;
 
-    @NotNull
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultPasswordEncoder.class);
+
     private final String encodingAlgorithm;
```

Comment:
I'm not sure why you remove the `@NotNull` annotation for the `encodingAlgorithm`. It's mandatory, isn't it?